src/ctapipe/instrument/tests/test_psf_model.py

```diff
@@ -7,26 +7,43 @@
 from ctapipe.instrument.optics import PSFModel
 
 
-def test_psf(example_subarray):
+@pytest.fixture(scope="session")
+def asymmetry_params():
+    return [0.49244797, 9.23573115, 0.15216096]
```

Comment:
Fixtures are not really meant for simple data like this. 

You could just enter them in the test itself.

Also: why these highly specific values just for a unit test? Wouldn't nice round values in the right order of magnitude be easier on the eyes?